src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -311,22 +334,22 @@ await _context.Entry(methodology.MethodologyParent)
 
             return status switch
             {
-                MethodologyStatus.Draft => _userService.CheckCanMarkMethodologyAsDraft(methodology),
-                MethodologyStatus.Approved => _userService.CheckCanApproveMethodology(methodology),
+                Draft => _userService.CheckCanMarkMethodologyAsDraft(methodology),
```

Comment:
I think there needs to be a change a few lines above this one to avoid skipping this check if the status is not changing. Since we now only call this method when going down the code  path of changing the status (as opposed to changing the details), we should always do this rather than return early.

This might need to remove the check on the ApproveSpecificMethodologyAuthorizationHandler which first checks that a methodology is not approved, but I still think it's better to do that and check they can approve it, rather than bypassing this check.

Reply:
This is fixed by #2769